### PR TITLE
[FLINK-33380][connectors/mongodb] Bump flink version on flink-connectors-mongodb with push_pr.yml and w…

### PR DESCRIPTION
**Pull request by gtk96**
…eekly.yml
[FLINK-33380](https://issues.apache.org/jira/browse/FLINK-33380) 

add new versions CI check in push_pr.yml and weekly.yml.

### CODE REVIEW

**Comment by gtk96** [comment]
@Jiabao-Sun There seems to be some conflict?

**Comment by Jiabao-Sun** [comment]
> @Jiabao-Sun There seems to be some conflict?

Hi @gtk96.

Try to exclude commons-compress from org.testcontainers:junit-jupiter

```xml
<dependency>
	<groupId>org.testcontainers</groupId>
	<artifactId>junit-jupiter</artifactId>
	<scope>test</scope>
	<exclusions>
		<exclusion>
			<groupId>org.apache.commons</groupId>
			<artifactId>commons-compress</artifactId>
		</exclusion>
	</exclusions>
</dependency>
```

**Comment by MartijnVisser** [comment]
> Try to exclude commons-compress from org.testcontainers:junit-jupiter

Or pin the version

**Comment by gtk96** [comment]
@Jiabao-Sun @MartijnVisser  ci pass https://github.com/gtk96/flink-connector-mongodb/actions/runs/6700848560 PTAL

File: pom.xml [hunk 1]
```diff
@@ -119,11 +119,11 @@ under the License.
...
-		<dependency>
-			<groupId>org.testcontainers</groupId>
-			<artifactId>junit-jupiter</artifactId>
-			<scope>test</scope>
-		</dependency>
+        <dependency>
+	       <groupId>org.testcontainers</groupId>
+	       <artifactId>junit-jupiter</artifactId>
+	       <scope>test</scope>
+        </dependency>
```
Comment (Jiabao-Sun) [inline pom.xml:126]: I think there is no need to modify the indentation symbol here.

Comment (gtk96) [reply pom.xml]: fix

File: .github/workflows/weekly.yml [hunk 2]
```diff
@@ -35,15 +35,15 @@ jobs:
...
         }, {
           flink: 1.16.2,
           branch: v1.0
         }, {
           flink: 1.17.1,
           branch: v1.0
-        }, {
-          flink: 1.18-SNAPSHOT,
-          branch: v1.0
         }]
```
Comment (Jiabao-Sun) [inline .github/workflows/weekly.yml:47]: I think we should't remove this lines here.

```js
        {
          flink: 1.16.2,
          branch: v1.0
        }, {
          flink: 1.17.1,
          branch: v1.0
        }, {
          flink: 1.18.0,
          branch: v1.0
        }, {
          flink: 1.19-SNAPSHOT,
          branch: v1.0
        }
```

Comment (gtk96) [reply .github/workflows/weekly.yml:47]: Does version 1.0 support 1.18 and 1.19?

File: .github/workflows/push_pr.yml [hunk 3]
```diff
@@ -25,7 +25,7 @@ jobs:
   compile_and_test:
     strategy:
       matrix:
-        flink: [1.17-SNAPSHOT, 1.18-SNAPSHOT]
+        flink: [1.17.1, 1.18.0, 1.19-SNAPSHOT]
```
Comment (Jiabao-Sun) [inline .github/workflows/push_pr.yml:28]:
```suggestion
        flink: [1.16-SNAPSHOT, 1.17-SNAPSHOT, 1.18-SNAPSHOT, 1.19-SNAPSHOT]
```

File: .github/workflows/weekly.yml (hunk 2 above)
Comment (Jiabao-Sun) [reply .github/workflows/weekly.yml:47]: For 1.19 we need pin the version in pom.xml.
I think we can cherry-pick this commit into v1.0 branch as well.

**Review by Jiabao-Sun — APPROVED** [review]
Thanks @gtk96 for the great work. LGTM.
cc @MartijnVisser

**Pull request merged**